.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "multibuild"]
+	path = multibuild
+	url = https://github.com/matthew-brett/multibuild.git
```

Comment:
As we discussed, it is not worth having this as a submodule, we can simply have a `git clone` in the `.travis` file

Reply:
Ok, updated and tested